designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt

```diff
@@ -298,15 +299,26 @@ internal fun designSwitcherDocName() = "DesignSwitcherDoc"
  *
  * Controls whether the Design Switcher will fetch updates to it's DesignDoc from Figma.
  *
- * This should only be changed temporarily. Do not commit a change to `false`.
+ * This should only be changed temporarily. Do not commit a change to `true`.
  */
-private const val DISABLE_LIVE_MODE = true
+private const val ENABLE_LIVE_MODE = false
 
-private fun getLiveMode(): LiveUpdateMode {
-    return if (DISABLE_LIVE_MODE) {
-        LiveUpdateMode.OFFLINE
+private var enableLiveModeForTesting = false
+
+internal fun enableLiveModeForTesting(enabled: Boolean) {
+    if (Build.FINGERPRINT == "robolectric") {
+        enableLiveModeForTesting = enabled
     } else {
+        // TODO: no exception for integration tests...
```

Comment:
Clean up the Todo? I think it's fine as is, it's certainly not an error that should be handled.

Reply:
Done.